linkerd/docs/config.md

```diff
@@ -273,6 +273,46 @@ aperture supports the following options (see [here][aperture] for option semanti
 
 heap does not support any options.
 
+
+#### Retries
+
+linkerd can automatically retry requests on certain failures (for
+example, connection errors).
+
+* *retries* -- Optional. A retry policy for all clients created by
+  this router.
+  * *budget* -- Optional. Determines _how many_ failed requests are
+    eligible to be retried.
+    * *minRetriesPerSec* -- Optional. The minimum rate of retries
+      allowed in order to accommodate clients that have just started
+      issuing requests as well as clients that do not issue many
+      requests per window. Must be non-negative and if `0`, then no
+      reserve is given. (Default: 10)
+    * *percentCanRetry* -- Optional. The percentage of calls that can
+      be retried. This is in addition to any retries allowed for via
+      `minRetriesPerSec`.  Must be >= 0 and <= 1000. As an example, if
+      `0.1` is used, then for every 10 non-retry calls , 1 retry will
+      be allowed. If `2.0` is used then every non-retry call, 2
+      retries. (Default: 0.2)
```

Comment:
I think this sentence is clearer as: "If 2.0 is used then every non-retry call will be allowed 2 retries."